config/index.js

```diff
@@ -459,12 +459,10 @@ var conf = convict({
       ],
       env: 'SIGNIN_CONFIRMATION_SUPPORTED_CLIENTS'
     },
-    forceEmailRegex: {
+    enabledEmailAddresses: {
```

Comment:
As the format has changed, I saw this as an opportunity to use a consistent name for both email address settings.